server/controllers/calls.js

```diff
@@ -95,19 +96,25 @@ function saveNewAttempt(contact_id, campaign_id, attempt_num) {
   return callsService.createNewAttempt(newCallParams);
 }
 
+function saveResponses(responses, call_id) {
+  return Promise.all(responses.map((resp) => {
+    const { question_id, response } = resp;
+    const responseParams = { call_id, question_id, response };
+    return responsesService.saveNewResponse(responseParams)
+      .then()
```

Comment:
should we consider letting the front end know this was successful?

Reply:
That's happening below!